### DISCUSSION TITLE
EBSD Image Quality (iq) property

### DISCUSSION THREAD

**Discussion by m-amrei** [mtex-toolbox/mtex#1479] [Feature Requests]
Hello, 
I have noticed that you have been using the iq property [here ](https://mtex-toolbox.github.io/CSLBoundaries.html) however there was no explanation about how to calculate it (I reckon that it is probably a vendor property). Would you please explain this a bit more?

Regards, 
Mohsen

**Comment by lfgmorales** [comment]
hi


look here


https://www.cambridge.org/core/journals/microscopy-and-microanalysis/article/ebsd-image-quality-mapping/5B6B1BFDCFFC031D1E317CE3282C029A


cheers


Luiz


Luiz F. G. Morales
ScopeM & Structural Geology and Tectonics Group
ETH Zürich
HPM C57.3

Otto-Stern-Weg 3

8093 Zürich

Switzerland

Phone: + 41 44 633 3746

<tel:+41446333746>E-mail: ***@***.***



"No trees were destroyed in the sending of this message, however, a significant number of electrons were terribly inconvenienced"


________________________________
From: Mohsen Mokhtabad Amrei ***@***.***>
Sent: 06 July 2022 14:54:14
To: mtex-toolbox/mtex
Cc: Subscribed
Subject: [mtex-toolbox/mtex] EBSD Image Quality (iq) property (Discussion #1479)


Hello,
I have noticed that you have been using the iq property here <https://mtex-toolbox.github.io/CSLBoundaries.html> however there was no explanation about how to calculate it (I reckon that it is probably a vendor property). Would you please explain this a bit more?

Regards,
Mohsen

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1479>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN6EMKBDJKX2WHEJHSTVSV6XNANCNFSM52ZVK56Q>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1479", "url": "https://github.com/mtex-toolbox/mtex/discussions/1479", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]
